aksparse/client/aksparse/sparse.py

```diff
@@ -442,48 +474,54 @@ def dot(self, input):
             temp_csr = self.to_csr()
             return temp_csr.vector_matrix_dot(input)
         if str(type(input)) == "<class ''>":
-            temp_csr = self.to_csr()
-            return temp_csr.matrix_matrix_dot(input)
+            raise NotImplementedError
+            # temp_csr = self.to_csr()
+            # return temp_csr.matrix_matrix_dot(input)
```

Comment:
If the old code is not needed, we should remove this.

Reply:
And again, deleted!